pkg/cmd/cli/describe/describer.go

```diff
@@ -534,6 +534,24 @@ func (d *ImageDescriber) Describe(namespace, name string, settings kctl.Describe
 	return describeImage(image, "")
 }
 
+func describeImageSignature(s imageapi.ImageSignature, out *tabwriter.Writer) error {
+	formatString(out, "\tName", s.Name)
+	formatString(out, "\tType", s.Type)
+	if s.IssuedBy == nil {
+		// FIXME: Make this constant
+		formatString(out, "\tStatus", "Unverified")
```

Comment:
@smarterclayton I thought we want to have clear message about "verified vs. unverified" image. Returning only conditions gives a person who is familiar with them clue about the verification status but I don't think it will make it obvious.